src/keys.rs

```diff
@@ -1288,4 +1292,22 @@ mod tests {
             assert_eq!(internal_ovk.0, tv.internal_ovk);
         }
     }
+
+    #[test]
+    fn zsa_issuance_auth_sig_test_vectors() {
```

Comment:
remove `zsa_` prefix

Reply:
Renamed to `issuance_auth_sig_test_vectors`.